Headers/AppKit/NSSplitViewItem.h

```diff
@@ -0,0 +1,118 @@
+/* Interface of class NSSplitViewItem
+   Copyright (C) 2020 Free Software Foundation, Inc.
+   
+   By: Gregory John Casamento
+   Date: Mon 20 Jul 2020 12:56:20 AM EDT
+
+   This file is part of the GNUstep Library.
+   
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2.1 of the License, or (at your option) any later version.
+   
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+   
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; if not, write to the Free
+   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110 USA.
+*/
+
+#ifndef _NSSplitViewItem_h_GNUSTEP_GUI_INCLUDE
+#define _NSSplitViewItem_h_GNUSTEP_GUI_INCLUDE
+
+#import <Foundation/NSObject.h>
+#import <AppKit/AppKitDefines.h>
+
+#if OS_API_VERSION(MAC_OS_X_VERSION_10_10, GS_API_LATEST)
+
+#if	defined(__cplusplus)
+extern "C" {
+#endif
+
+enum
+{
+ NSSplitViewItemBehaviorDefault,
+ NSSplibViewItemBehaviorSidebar,
+ NSSplitViewItemBehaviorContentList
+};
+typedef NSInteger NSSplitViewItemBehavior;
+
+enum
+{
+ NSSplitViewItemCollapseBehaviorDefault,
+ NSSplitViewItemCollapseBehaviorPreferResizingSiblingsWithFixedSplitView,
+ NSSplitViewItemCollapseBehaviorPreferResizingSplitViewWithFixedSiblings,
+ NSSplitViewItemCollapseBehaviorUseConstraints
+};
+typedef NSInteger NSSplitViewItemCollapseBehavior;
+
+enum
+{
+ NSTitlebarSeparatorStyleAutomatic,
+ NSTitlebarSeparatorStyleLine,
+ NSTitlebarSeparatorStyleNone,
+ NSTitlebarSeparatorStyleShadow
+};
+typedef NSInteger NSTitlebarSeparatorStyle;
+
+@class NSViewController;
+  
+@interface NSSplitViewItem : NSObject <NSCoding, NSCopying>
```

Comment:
Apple only list NSCoding here in the documentation.

Reply:
fixed